### PR TITLE
commander: move RootFingerprintRequest to Rust

### PR DESCRIPTION
**Pull request by benma**

### CODE REVIEW

File: src/rust/bitbox02-rust/src/hww/api/rootfingerprint.rs [hunk 1]
```diff
@@ -0,0 +1,30 @@
+// Copyright 2020 Shift Crypto AG
```
Comment (x1ddos) [inline src/rust/bitbox02-rust/src/hww/api/rootfingerprint.rs:1]: 2021?

**Review by x1ddos — APPROVED** [review]
LGTM

File: src/rust/bitbox02-rust/src/hww/api/rootfingerprint.rs (hunk 1 above)
Comment (benma) [reply src/rust/bitbox02-rust/src/hww/api/rootfingerprint.rs:1]: I wrote it in 2020 :)

Comment (benma) [reply src/rust/bitbox02-rust/src/hww/api/rootfingerprint.rs]: Done!

**Pull request merged**